roles/velero/tasks/main.yml

```diff
@@ -78,11 +78,24 @@
     state: "{{ velero_state }}"
     definition: "{{ lookup('template', 'velero.yml.j2') }}"
 
-- name: "Create BackupStorageLocations"
+- name: "Create BackupStorageLocations if aws is configured for velero"
```

Comment:
NIT: change log `Create AWS BackupStorageLocation`

Reply:
Done, Updated the log.